Entities/Scene.h

```diff
@@ -686,33 +686,25 @@ EntityAllocation(Scene)
     bool CleanOrphanPixel(int posX, int posY, NeighborDirection checkingFrom = NODIR, int team = Activity::TeamOne);
 
 
-//////////////////////////////////////////////////////////////////////////////////////////
-// Method:          GetDimensions
-//////////////////////////////////////////////////////////////////////////////////////////
-// Description:     Gets the total dimensions (width and height) of the scene, in pixels.
-// Arguments:       None.
-// Return value:    A Vector describing the scene dimensions.
-
+    /// <summary>
+    /// Gets the total dimensions (width and height) of the scene, in pixels.
+    /// </summary>
+    /// <returns>A Vector describing the scene dimensions.</returns>
+	/// <remarks>This will attempt to get the dimensions from the bitmap if it is loaded into memory. If not, will fall back to the info that was read directly from the file on disk at creation.</remarks>
```

Comment:
Bit of inconsistent space/tab stuff here I guess. Not really worth worrying about

More importantly, do we really need the remark? Doesn't really matter to anything calling it, does it?

All this applies to other methods as well.